### PR TITLE
feat: delete doc mutation

### PR DESCRIPTION
**Pull request by e-lobo**

### CODE REVIEW

File: frappe_graphql/utils/resolver.py [hunk 1]
```diff
@@ -131,6 +132,22 @@ def save_doc_resolver(obj: Any, info: GraphQLResolveInfo, **kwargs):
...
 
+def delete_doc_resolver(obj: Any, info: GraphQLResolveInfo, **kwargs):
+    doctype = kwargs["doctype"]
+    name = kwargs["name"]
+    success  = False
+    try:
+        doc = frappe.get_doc(doctype, name)
+        doc.delete()
+        success = True
+    except Exception as e:
```
Comment (fahimalizain) [inline frappe_graphql/utils/resolver.py]: @e-lobo shouldn't we throw the error as it is ? and return only if successful ? @assemmarwan

Comment (e-lobo) [reply frappe_graphql/utils/resolver.py]: @faztp12 @assemmarwan there will be `_server_messages` for perm or link validation errors.. but i can `frappe.msgprint(frappe.get_traceback())` if needed, should i?

Comment (fahimalizain) [reply frappe_graphql/utils/resolver.py]: i think _server_messages shouldnt be relied on if we are to follow the graphql spec.

Comment (e-lobo) [reply frappe_graphql/utils/resolver.py]: @faztp12 what do you suggest?

Comment (fahimalizain) [reply frappe_graphql/utils/resolver.py]: @e-lobo simply throw the exception as it is, no try catch. what do you think ?

Comment (e-lobo) [reply frappe_graphql/utils/resolver.py]: @faztp12 done 👍

**Comment by assemmarwan** [comment]
@e-lobo Please update the README to include the delete mutation with an example

**Comment by e-lobo** [comment]
> @e-lobo Please update the README to include the delete mutation with an example

done

**Review by fahimalizain — APPROVED** [review]
👌👌

**Pull request merged**